### PR TITLE
feat: Add Arbitrum_RescueAdapter helper scripts

### PR DESCRIPTION
**Pull request by nicholaspai**
To aid with future usage of this adapter

### CODE REVIEW

File: contracts/chain-adapters/Arbitrum_Adapter.sol [hunk 1]
```diff
@@ -17,6 +17,17 @@ interface ArbitrumL1InboxLike {
         uint256 gasPriceBid,
         bytes calldata data
     ) external payable returns (uint256);
+
+    function createRetryableTicketNoRefundAliasRewrite(
```
Comment (mrice32) [inline contracts/chain-adapters/Arbitrum_Adapter.sol:21]: nit: should we use unsafeCreateRetryableTicket instead (I think the signature is the same)?

According to the [current Inbox implementation on mainnet](https://etherscan.io/address/0x5aed5f8a1e3607476f1f81c3d8fe126deb0afe94#code), createRetryableTicketNoRefundAliasRewrite is deprecated in favor of unsafeCreateRetryableTicket.

**Review by mrice32 — COMMENTED** [review]
Looks good! Only one minor nit (note: since this is already deployed, we can go ahead and use the existing implementation for this operation, but I think we should update the contract in the repo with the non-deprecated method).

File: contracts/chain-adapters/Arbitrum_Adapter.sol (hunk 1 above)
Comment (nicholaspai) [reply contracts/chain-adapters/Arbitrum_Adapter.sol:21]: This is a great call. `createRetryableTicketNoRefundAliasRewrite` calls `unsafeCreateRetryableTicket` so I agree we don't need to change deployed version

**Comment by nicholaspai** [comment]
This adapter worked!
- L2 txn: https://arbiscan.io/tx/0xe01150352f64d34511abcafe60e834de26f5194907882670b32135ee301866cd
- L1 txn: https://etherscan.io/tx/0x33c315881a73b63dc62a175bc86830e48b461797c4c87f8590376572e98e43cd

**Review by pxrl — COMMENTED** [review]
No comments from me :+1:.

**Review by mrice32 — APPROVED** [review]
LGTM!

**Pull request merged**